ambari-server/src/main/java/org/apache/ambari/server/stack/StackManager.java

```diff
@@ -380,6 +403,9 @@ public ExtensionInfo getExtension(String name, String version) {
    * @return true if all of the repo update tasks have completed; false otherwise
    */
   public boolean haveAllRepoUrlsBeenResolved() {
+    if(stackContext == null) {
```

Comment:
Should never be null.  How is that happening?  Even if there's no directory for a stack (how?) the context should be available.